### PR TITLE
Implement physical attributes NOMIS to DPS synchronisation

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt [hunk 1]
```diff
@@ -1,36 +1,439 @@
...
+                  ),
+                ),
+              ),
+            ),
+          ),
+        )
+
+        sendPhysicalAttributesChangedEvent()
+
+        verifyResults(type = "ignored", reason = "New physical attributes are empty")
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt:106]: Question; are physical attributes not copied by NOMIS for new bookings (like it does for Alerts)?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt [hunk 2]
```diff
@@ -1,36 +1,439 @@
...
+            assertThat(it["offenderNo"]).isEqualTo("A1234AA")
+            assertThat(it["bookingId"]).isEqualTo("12345")
+            assertThat(it["error"]).contains("Bad Request")
+          },
+          isNull(),
+        )
+      }
+
+      @Test
+      fun `should put message on DLQ if booking has no physical attributes`() = runTest {
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt:405]: why is this an error?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt (hunk 1 above)
Comment (mikehalmamoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt:106]: They are yes. This test is for a brand new booking - if you go into the physical info NOMIS screen it creates this "empty" record which makes no sense to synchronise.

**Review by andymarke — APPROVED** [review]
lgtm

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt (hunk 2 above)
Comment (mikehalmamoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonSyncIntTest.kt:405]: We shouldn't be receiving events for bookings without any physical attributes - but I chose not to assume that in the code, so had to handle the error scenario. Should never happen.

**Pull request merged**